lib/linguist/languages.yml

```
@@ -2353,6 +2353,19 @@ Go Module:
   tm_scope: go.mod
   ace_mode: text
   language_id: 947461016
+Go+:
+  type: programming
+  color: "#00A2DE"
+  aliases:
+  - gop
+  - goplus
+  extensions:
+  - ".gop"
+  tm_scope: source.gop
+  ace_mode: go+
+  codemirror_mode: go+
+  codemirror_mime_type: text/x-go+
```

Likewise, CodeMirror doesn't support Go+ either, so it's better to just reuse Go's CodeMirror mode:

```suggestion
  codemirror_mode: go
  codemirror_mime_type: text/x-go
```